libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -845,6 +845,8 @@ bool _IotMqtt_IncrementConnectionReferences( _mqttConnection_t * pMqttConnection
     if( disconnected == false )
     {
         ( pMqttConnection->references )++;
+
+        /* coverity[misra_c_2012_directive_4_6_violation] */
```

Comment:
'coverity' needs to be added in lexicon.txt file

Reply:
When adding annotations to disable MISRA rules for a line, please put a comment saying why.

Reply:
Added to the atomics folder. Already present in mqtt.